### ISSUE TITLE
Switch to mbdTLS instead as LibreSSL

### ISSUE THREAD

**Issue by GoetzGoerisch** [umati/Sample-Server#1208]
Switch back to using mbdTLS after this is now able to generate certificates in the context of open62541.
https://github.com/open62541/open62541/pull/6145

**Comment by ainglessi** [comment]
UA_CreateCertificate is not yet implemented when using Mbed TLS: https://github.com/open62541/open62541/issues/6874

**Comment by GoetzGoerisch** [comment]
@ccvca is this new, I though we had this already in the past but needed to use LibreSSL for certificate validation.

**Comment by ccvca** [comment]
No, the certificate generation was the reason for switching to LibreSSL. In the past, mbdTLS does not support the required fields in the certificate. 
But it looks like the current version of open62541 does support it:
https://github.com/open62541/open62541/pull/6535

**Comment by GoetzGoerisch** [comment]
UA_CreateCertificate is now included in `develop` as it was backportet to open62541-1.4.
